packages/web3-eth-contract/src/contract.ts

```diff
@@ -964,6 +965,21 @@ export class Contract<Abi extends ContractAbi>
 			: value;
 	}
 
+	public decodeMethodData(data: HexString): DecodedParams & { __method__: string } {
```

Comment:
pls add this in docs under generated methods https://github.com/web3/web3.js/blob/67d8a74386047de866456badb5ae3dae7cd4a9e6/packages/web3-eth-contract/src/contract.ts#L238 like https://docs.web3js.org/libdocs/Contract#generated-methods
Thanks